KTL.js

```diff
@@ -5619,7 +5621,28 @@ function Ktl($, appInfo) {
                 }
             }
         }
-
+        function makeTableHeaderSticky(viewId, keywords, data) {
+            const kw = '_sth';
+            let numOfRecords = 10;
+            let viewHeight = 800;
+            if (keywords[kw] && keywords[kw][0]?.params && keywords[kw][0]?.params[0].length) {
+                numOfRecords = keywords[kw][0].params[0][0] ? keywords[kw][0].params[0][0] : numOfRecords;
+                if (keywords[kw][0].params[0][1]) viewHeight = keywords[kw][0].params[0][1];
+                
+            }
+            ktl.views.stickTableHeaders(viewId, data, numOfRecords, viewHeight);
```

Comment:
You might need to move this in your `if`. Otherwise, it will be run with or without the keyword. Same for `5644 -             ktl.views.stickFirstColumn(viewId, stickyBkgdColor);`